app/controllers/deliveries_controller.rb

```diff
@@ -18,7 +16,7 @@ def new
 
   def create
     @delivery = Delivery.new(params[:delivery])
-    
```

Comment:
My editor does this automatically such that the diff gets cluttered in this way. Should I turn it off and rebase?